cleanlab_studio/studio/studio.py

```diff
@@ -144,3 +144,60 @@ def apply_corrections(self, cleanset_id: str, dataset: Any) -> Any:
             corrected_ds[label_column] = joined_ds["__cleanlab_final_label"]
             corrected_ds = corrected_ds[joined_ds["action"] != "exclude"]
             return corrected_ds
+
+    def create_project(
+        self,
+        dataset_id: str,
+        project_name: str,
+        *,
+        tasktype: str = "multi-class",
+        modality: Optional[str] = None,
+        modeltype: str = "regular",
+        label_column: Optional[str] = None,
+        feature_columns: Optional[List[str]] = None,
+        text_column: Optional[str] = None,
+    ) -> str:
+        dataset_details = api.get_dataset_details(self._api_key, dataset_id)
+
+        if modality is None:
+            print("Modality not supplied. Using dataset modality")
+            modality = dataset_details["modality"]
```

Comment:
thought: we want to deprecate dataset modality in the near future, and make modality a project-only construct. because of this, I don't think we want to make modality optional.

Reply:
Ah good call. That makes sense to me